src/Button/Button.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+require('./Button.less');
+
+class Button extends React.Component {
```

Comment:
@osdiab I already have a TODO to add docs to the README--are you suggesting additional documentation as comments in the code? I'd be worried about keeping these in sync.

Agreed that we should add `propTypes`